app/src/main/res/layout/activity_zim_host.xml

```diff
@@ -34,13 +34,13 @@
   <Button
     android:id="@+id/startServerButton"
     android:layout_width="match_parent"
-    android:layout_height="wrap_content"
+    android:layout_height="60dp"
     android:layout_marginStart="8dp"
     android:layout_marginLeft="8dp"
     android:layout_marginEnd="8dp"
     android:layout_marginRight="8dp"
     android:layout_marginBottom="8dp"
-    android:background="@color/startServerGreen"
+    android:backgroundTint="@color/startServerGreen"
```

Comment:
This also doesn't make sense in light of the original ticket, red wasn't showing up also? Have you tried this in light theme as well?

Reply:
yes @macgills I updated in the PR.

Reply:
hmmm @Frans-Lukas can you test this? Seems funky, right? I don't know how your device could have affected it but I want to remove the possibility

Reply:
Hmm, as long as the color updates dynamically. Will check this weekend.

Reply:
@macgills @gouri-panda This works fine for me 👍